scripts/build/Platform/Windows/build_config.json

```diff
@@ -449,12 +449,24 @@
         "CMAKE_NATIVE_BUILD_ARGS": "/m /nologo"
     }
   },
+  "install_mono_release": {
+    "TAGS": [],
+    "COMMAND": "build_windows.cmd",
+    "PARAMETERS": {
+      "CONFIGURATION": "release",
+      "OUTPUT_DIRECTORY": "build\\windows_mono",
+      "CMAKE_OPTIONS": "-G \"Visual Studio 16 2019\" -DCMAKE_SYSTEM_VERSION=10.0 -DLY_MONOLITHIC_GAME=TRUE -DLY_DISABLE_TEST_MODULES=TRUE",
```

Comment:
2019 not 2022?

Reply:
I don't know how this got into the PR, not related

Reply:
Confirmed all these diffs are already in the point release